### PR TITLE
Fixed Dark Mode Bugs

### PR DESCRIPTION
**Pull request by rileyhawk1417**
After re-forking from the main branch. Managed to update the files and fix the bugs.

### CODE REVIEW

File: frontend/app_flowy/lib/workspace/domain/edit_action/app_edit.dart [hunk 1]
```diff
@@ -21,9 +21,9 @@ extension AppDisclosureExtension on AppDisclosureAction {
   Widget get icon {
     switch (this) {
       case AppDisclosureAction.rename:
-        return svg('editor/edit');
+        return svg('editor/edit', color: const Color(0xffe5e5e5));
```
Comment (appflowy) [inline frontend/app_flowy/lib/workspace/domain/edit_action/app_edit.dart:24]: 0xffe5e5e5 works for light mode too?

File: frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/app/section/item.dart [hunk 2]
```diff
@@ -44,7 +44,7 @@ class ViewSectionItem extends StatelessWidget {
             onTap: () => onSelected(context.read<ViewBloc>().state.view),
             child: FlowyHover(
               config: HoverDisplayConfig(hoverColor: theme.bg3),
-              builder: (_, onHover) => _render(context, onHover, state),
+              builder: (_, onHover) => _render(context, onHover, state, theme.textColor),
```
Comment (appflowy) [inline frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/app/section/item.dart:47]: I think textColor is only for text. Here would be icon color?

File: frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/app/section/item.dart [hunk 3]
```diff
@@ -53,9 +53,9 @@ class ViewSectionItem extends StatelessWidget {
     );
   }
 
-  Widget _render(BuildContext context, bool onHover, ViewState state) {
+  Widget _render(BuildContext context, bool onHover, ViewState state, Color icon) {
```
Comment (appflowy) [inline frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/app/section/item.dart:56]: Replacing Color icon  with Color iconColor would be better

File: frontend/app_flowy/packages/flowy_infra/lib/theme.dart [hunk 4]
```diff
@@ -141,9 +141,9 @@ class AppTheme {
           primaryVariant: main2,
           secondary: main2,
           secondaryVariant: main2,
-          background: bg1,
+          background: surface,
```
Comment (appflowy) [inline frontend/app_flowy/packages/flowy_infra/lib/theme.dart:144]: Why replace the background from bg1 to surface?

File: frontend/app_flowy/packages/flowy_infra/lib/theme.dart [hunk 5]
```diff
@@ -141,9 +141,9 @@ class AppTheme {
...
           surface: surface,
-          onBackground: bg1,
+          onBackground: surface,
```
Comment (appflowy) [inline frontend/app_flowy/packages/flowy_infra/lib/theme.dart:146]: Make sure these colors you're filled also work on light mode.

Comment (rileyhawk1417) [reply frontend/app_flowy/packages/flowy_infra/lib/theme.dart:146]: Was following Bran's Color Scheme in the [figma](https://www.figma.com/file/Ty2OmskN8oKgj2fLrqGer9/Appflowy.io-(Community)?node-id=2%3A307) document. With the naming conventions was not sure which color goes where.

File: frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/top_bar.dart [hunk 6]
```diff
@@ -4,24 +4,29 @@ import 'package:flowy_infra/image.dart';
 import 'package:flowy_infra_ui/style_widget/icon_button.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:provider/provider.dart';
```
Comment (MikeWallaceDev) [inline frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/top_bar.dart:7]: This line causes a warning, please fix

File: frontend/app_flowy/packages/flowy_infra/lib/theme.dart [hunk 7]
```diff
@@ -93,9 +93,9 @@ class AppTheme {
 
       case ThemeType.dark:
         return AppTheme(isDark: true)
-          ..surface = _black
-          ..hover = _main2
-          ..selector = _black
+          ..surface = const Color(0xff292929)
```
Comment (MikeWallaceDev) [inline frontend/app_flowy/packages/flowy_infra/lib/theme.dart:96]: This file has a warning on line 13. Please fix.

File: frontend/app_flowy/lib/workspace/domain/image.dart [hunk 8]
```diff
@@ -8,9 +8,9 @@ AssetImage assetImageForViewType(ViewType type) {
 }
 
 extension SvgViewType on View {
-  Widget thumbnail() {
+  Widget thumbnail(Color iconColor) {
```
Comment (appflowy) [inline frontend/app_flowy/lib/workspace/domain/image.dart:11]: Using the optional params would be better?

`Widget thumbail({Color? iconColor})`

File: frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/top_bar.dart (hunk 6 above)
Comment (appflowy) [reply frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/top_bar.dart:7]: I think the CI should block the codes with warnings?

File: frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/app/section/item.dart (hunk 2 above)
Comment (rileyhawk1417) [reply frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/app/section/item.dart]: Fixed and created a new value "iconColor"

File: frontend/app_flowy/packages/flowy_infra/lib/theme.dart (hunk 7 above)
Comment (rileyhawk1417) [reply frontend/app_flowy/packages/flowy_infra/lib/theme.dart:97]: Removed the warnings

**Pull request merged**